pkg/server/server.go

```diff
@@ -397,6 +397,7 @@ func (s *Server) newEndpointsServer(ctx context.Context, catalog catalog.Catalog
 		CacheReloadInterval:          s.config.CacheReloadInterval,
 		EventsBasedCache:             s.config.EventsBasedCache,
 		PruneEventsOlderThan:         s.config.PruneEventsOlderThan,
+		SQLTransactionTimeout:        s.config.SQLTransactionTimeout,
```

Comment:
it'd be nice to unit test, considering lack thereof caused the issue

Reply:
Hey @amoore877 That was my first thought too. Unfortunately I dont know where to work in a unit test here as the fields just end up as unexported fields in the `endpoints` object.

Reply:
ah, and there's no way to easily test with a running module and infer behavior from config I guess. sad times.

the most I've done sometimes in such cases at least verify those unexported fields (which would be accessible in a same-package test) are set as expected. it's one of those kinds of tests that seems silly at first, but catches simple mistakes.

better though would be that behavioral check, but that seems hard compared to the effort of the fix :)